.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+name: Release google-java-format
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "version number for this release."
+        required: true
+
+jobs:
+  build-maven-jars:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Setup Signing Key
+        run: |
+          cat <(echo -e "${{ secrets.GPG_SIGNING_KEY }}") | gpg --batch --import
+          gpg --list-secret-keys --keyid-format LONG
+          
+      - name: Checkout
+        uses: actions/checkout@2.4.0
+
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: 17
+          distribution: 'zulu'
+          cache: 'maven'
+     
+      - name: Bump Version Number
+        env:
+          NEW_VERSION: ${{ github.events.input.version }}
+        run: |
+          mvn versions:set versions:commit -DnewVersion="${NEW_VERSION}"
+          git ls-files | grep 'pom.xml$' | xargs git add
+          git commit -m "Release google-java-format ${NEW_VERSION}"
+          git push
+
+      - name: Build Jars
+        run: mvn clean verify gpg:sign -DskipTests=true -Dgpg.passphrase="${{ secrets.GPG_PASSPHRASE }}"
```

Comment:
I think once we're doing the actual `mvn deploy` it will produce the same local jars to we probably don't need tow separate checks for this. Also we should run the tests on at least one of the invocations (by omitting the `-DskipTests=true`), but this will be faster for manually testing the workflow

Reply:
SGTM